src/devtools/views/Components/InspectedElementTree.js

```diff
@@ -37,14 +38,17 @@ export default function InspectedElementTree({
       <div className={styles.InspectedElementTree}>
         <div className={styles.HeaderRow}>
           <div className={styles.Header}>{label}</div>
-          {!isEmpty && (
+          <div style={{ visibility: isEmpty ? 'hidden' : '' }}>
```

Comment:
Without this, `props` label jumps between items that have props and items that don't have them because the "copy" button influences the section height.